samples: bluetooth: mesh: remove pm from mesh dfu samples (#28521)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/dfu/target/README.rst

.. include:: /includes/build_and_run.txt

.. note::

>    For the ``nrf52840dongle/nrf52840`` platform use the merged hex file :file:`/build/merged_nrf52840dongle_nrf52840.hex` then follow the instruction from the :zephyr:board:`nrf52840dongle` documentation on how to create and flash a DFU package using ``nrfutil``.

Reviewer: ```suggestion
   For the ``nrf52840dongle/nrf52840`` board target, use the merged :file:`/build/merged_nrf52840dongle_nrf52840.hex` file and follow the instructions in the :zephyr:board:`nrf52840dongle` documentation on how to create and flash a DFU package using ``nrfutil``.
```
